images/kubectl-build-deploy-dind/helmcharts/python-persistent/templates/deployment.yaml

```
@@ -0,0 +1,80 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "python-persistent.fullname" . }}
+  labels:
+    {{- include "python-persistent.labels" . | nindent 4 }}
+  annotations:
+    {{- include "python-persistent.annotations" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "python-persistent.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "python-persistent.labels" . | nindent 8 }}
+      annotations:
+        {{- include "python-persistent.annotations" . | nindent 8 }}
```

also, if we add the configMapSha as an annotation, it will redeploy on a configMap update
```suggestion
        {{- include "python-persistent.annotations" . | nindent 8 }}
        lagoon.sh/configMapSha: {{ .Values.configMapSha | quote }}
```